docs/specs/aerogear-client-pipe.html

```diff
@@ -734,7 +734,8 @@
 </div>
 <div id="content">
 <div class="paragraph"><p>---
-layout: basic

```

Comment:
I believe someone just accidentally commited HTML generated from `aerogear-client-pipe.asciidoc`. I'll remove it.